ci-operator/jobs/openshift/assisted-service/openshift-assisted-service-release-ocm-2.5-presubmits.yaml

```diff
@@ -1363,3 +1363,60 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )unit-test,?($|\s.*)
+  - agent: kubernetes
+    always_run: false
```

Comment:
Why?

Reply:
it means it will skip on those cases (and will run for all other cases):
```
skip_if_only_changed: ^docs/|^sites/|\.md$|^(LICENSE|OWNERS|OWNER_ALIASES)$
```
(from line 1381)

I agree with you it's pretty confusing, I would prefer if prow would made it more intuitive: https://github.com/kubernetes/test-infra/blob/master/prow/jobs.md#triggering-jobs-based-on-changes

Reply:
Ah I see